modules/tinymce/src/core/main/ts/Rtc.ts

```diff
@@ -272,6 +273,45 @@ const makeRtcAdaptor = (tinymceEditor: Editor, rtcEditor: RtcRuntimeApi): RtcAda
   };
 };
 
+const makeNoopAdaptor = (): RtcAdaptor => ({
+  undoManager: {
+    beforeChange: () => {},
+    addUndoLevel: () => null,
```

Comment:
We should maybe pull this out into a constant and use something like this? Atm this will cause a lot more bloat than a single variable once minified.
```
const nul = Fun.constant(null);
```

(Can't think of a better name than `nul` atm sorry)

Reply:
That is fine. Thanks.